### PR TITLE
Send email report with broken links

### PR DESCRIPTION
**Pull request by Debakel**

### CODE REVIEW

**Comment by andybak** [comment]
Let me see if I understand this correctly:
1. You can (optionally) add a alert_mail_field property to linklists.py
2. If you do this email address is stored in each Link instance
3. A separate management command can be used to email a report to each person

Assumptions:
1. If you don't run the management command then everything works the same as without this feature
2. If you don't add the alert_mail_field property then everything works the same as without this feature.

Even though I don't have a use for it I guess I can see how it could be useful. 

I do wonder if we really need a new database field. Each Link instance knows it's model class and from that it should be possible to match that up with the linklist and thus the email address.

File: linkcheck/__init__.py [hunk 1]
```diff
@@ -179,6 +179,16 @@ def images(self, obj):
 
         return urls
 
+    def alert_mail(self, obj):
+        from utils import rgetattr
```
Comment (claudep) [inline linkcheck/__init__.py]: Use a relative import (`from .utils ...`)

File: linkcheck/south_migrations/0003_alert_mail.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
```
Comment (claudep) [inline linkcheck/south_migrations/0003_alert_mail.py]: South migrations are only historical and will be removed soon. Don't touch them any more...

File: linkcheck/templates/linkcheck/mail_report.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+{% block content %}
+Broken links where found:
+
+    {% for link in links %}
+    Object: {{ link.content_object }}
+    Url: {{ link.url }}
+
+    {% endfor %}
+
+{% endblock %}
```
Comment (claudep) [inline linkcheck/templates/linkcheck/mail_report.html:21]: I'm afraid this HTML template is missing markup...

**Comment by andybak** [comment]
@Debakel did you see my comments here: https://github.com/DjangoAdminHackers/django-linkcheck/pull/64#issuecomment-260914733 - I was hoping for a discussion!

**Pull request closed without merging**